docarray/array/storage/elastic/backend.py

```diff
@@ -159,10 +159,32 @@ def _build_client(self):
         client.indices.refresh(index=self._config.index_name)
         return client
 
-    def _send_requests(self, request) -> List[Dict]:
+    def _send_requests(self, request, **kwargs) -> List[Dict]:
         """Send bulk request to Elastic and gather the successful info"""
+        bulk_config = {}
+
+        # for backward compatibility
+        if 'chunk_size' not in kwargs:
+            bulk_config['chunk_size'] = self._config.batch_size
+
+        # Only accept several params for es parallel bulk operation
+        accepted_config = [
```

Comment:
this is not needed. Leave the freedom to user. We do not want to mantain this for every backend. We let the power to the user. We should add the corresponding docstring and that is.

Reply:
noted